modules/bridge/src/main/ts/ephox/bridge/components/view/ViewButton.ts

```diff
@@ -61,10 +73,16 @@ export interface ViewToggleButton extends BaseButton<ViewToggleButtonApi> {
 }
 export interface ViewButtonsGroup {
   type: 'group';
-  buttons: Array<ViewNormalButton | ViewToggleButton>;
+  buttons: Array<ViewNormalButton | ViewToggleButton | ViewLabel>;
```

Comment:
Having label mixed in with button components like this confuses me. Would renaming it help? For example:
```typescript
export interface ViewComponentsGroupSpec {
  type: 'group';
  items: Array<ViewNormalButtonSpec | ViewToggleButtonSpec | ViewLabelSpec>;
 }
 ```

Reply:
Sadly this wasn't picked up when views were created a year ago. We're now stuck with it (unless we want to put a breaking change in the next major release).